packages/tasks/src/model-libraries.ts

```diff
@@ -212,6 +212,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/cartesia-ai/cartesia_mlx",
 		snippets: snippets.cartesia_mlx,
 	},
+	"clip-scope": {
+		prettyLabel: "CLIP-ViT-L-scope",
+		repoName: "CLIP-ViT-L-scope",
+		repoUrl: "https://github.com/Lewington-pitsos/vitsae", 
+		filter: false,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
By doing so, any download of any pt file in the model will count as a new download. This means that if a user downloads the full repo, we will count it as X downloads (~80?). It's ok to do so if you think users will only download files one by one depending on their use case.

Another problem is that all downloads are tracked as 1 metric and you won't be able to know if users are more interested by a layer or another. A solution for this would be to create N models on the Hub (1 per layer for instance) and then create a [Collection](https://huggingface.co/collections) so group them all. For example you can find all models related to the Gemma scope release in this collection: https://huggingface.co/collections/google/gemma-scope-release-66a4271f6f0b4d4a9d5e04e2. A benefit of having a collection of models is that downloads will be tracked for each model individually, giving you more insights on the usage of your library. 

If you worry about the cost of maintaining all these repos, I'd recommend automating the upload and consistency between model cards using a script based on [`huggingface_hub`](https://huggingface.co/docs/huggingface_hub/index). Let me know what you think :)

Reply:
> In terms of tracking, these models are all derived from the same training process and a common use case for them should be to compare how they perform against one another. In a sense collectively they comprise a single tool. https://huggingface.co/google/gemma-scope-2b-pt-res from which we took inspiration takes a similar approach and places large numbers of related models together as a single "model".

Makes sense to keep `countDownloads: path_extension:"pt",` in that case!